docs/source/use_cases/index.rst

```diff
@@ -1,25 +1,96 @@
+
+Using Hyperledger Iroha each education or medical certificate can be verified that it was given by certification authorities. Immutability and clear validation rules provide transparency to health and education significantly reducing the usage of fake certificates.  
 
 Cross-border asset transfers
 ----------------------------
 
+Hyperledger Iroha provides fast and clear trade and settlement rules using multi-signature accounts and atomic exchange.   
+Asset management is easy as in centralized systems while providing necessary security guarantees.   
+ 
+By simplifying the rules and commands need to create and transfer assets, we lower the barrier to entry, while at the same time maintaining high-security guarantees.  
+
+Financial applications 
+----------------------------
+
+Hyperleger Iroha can be very useful in the auditing process. Each information in the system has defined access rules. These access rules can be defined at different levels: user-level, domain-level or system-level.  At the user-level privacy rules for a specific individual are defined. If access rules are defined determined at domain or system level, they are affecting all user in the domain. In Hyperledger Iroha we provide convenient role-based access control, where each role has specific permissions. 
+
+For example, a role of the auditor can have permissions to access the information of a specific user in the domain without bothering the user. To reduce the probability of account hijacking and prevent the auditor from sending malicious queries, the auditor is typically defined as a multi-signature account, meaning that auditor can make queries only having signatures from multiple separate identities.    
+
+Transactions can be traced with a with a local database. Using Iroha-API auditor can query and perform analytics on the data,  execute specific audit software. Hyperledger Iroha supports different scenarios for deploying analytics software:  on a local computer, or execute code on specific middleware.  
+This approach allows analyzing Big Data application with Hadoop, Apache etc.  Hypeledger Iroha serves as a guarantor of data integrity and privacy (due to the query permission restriction). 
+
+
 Identity management
 -------------------
 
-Medical applications
---------------------
+Hyperledger Iroha has an intrinsic support for identity management. Each user in the system has a uniquely identified account with personal information, and each transaction is signed and associated by a certain user.  This makes Hyperledger Iroha perfect for various application with KYC (Know Your Customer). 
+For example, insurance companies can benefit from querying the information of user’s transaction without worrying about the information truthfulness.  The user can benefit from storing personal information on a blockchain too since authenticated information will reduce the time of claims processing.   a
```

Comment:
2 spaces

Reply:
>   a

⭕️

Reply:
> The user can benefit from storing personal information on a blockchain too since authenticated information will reduce the time of claims processing.

please split